fahclient/templates/statefulset.yaml

```
@@ -26,7 +26,7 @@ spec:
       initContainers:
       - name: init
         image: alpine:latest
-        command: ['sh', '-c', "mkdir -p /fah && cp -f /tmp/fahclient/config.xml /fah/config.xml && chown -R {{ .Values.securityContext.runAsUser }}:{{ .Values.securityContext.runAsGroup }} /fah"]
+        command: ['sh', '-c', "mkdir -p /fah && cp -f /tmp/fahclient/config.xml /fah/config.xml"]
```

I actually introduced this bug but in the case where you don't want to run the process as root, we do need this chown so I suggest updating it to:
```suggestion
{{- $runAsUser:= .Values.securityContext.runAsUser | default .Values.securityContext.runAsGroup -}}
{{- $runAsGroup := .Values.securityContext.runAsGroup | default .Values.securityContext.runAsUser -}}
{{- if or .runAsUser .runAsGroup  -}}
{{- $chownCmd:= " && chown -R {{ .runAsUser }}:{{ .runAsGroup  }} /fah" -}}
{{- else -}}
{{- $chownCmd:= "" -}}
{{- end }}
        command: ['sh', '-c', {{ cat "mkdir -p /fah && cp -f /tmp/fahclient/config.xml /fah/config.xml" .chownCmd }}"]
```
Sounds like you run as root (which is not ideal) so if you are willing, you can let me know if this suggestion works with that.

I don't run these pods as root, but I'll report back after testing root and non-root and make a PR to fix.

---

````
kubectl exec -it fah-fahclient-0 -n fah -- whoami
whoami: cannot find name for user ID 9999
````
Not running as root.

I'm not changing the chart much on my own, here's my whole `values.yaml` that I use to customize install
````
replicaCount: 3
resources:
  limits:
    cpu: 2000m
  requests:
    cpu: 2000m
priorityClassName: "low-priority"
````

---

Check out my comment in Issue #34  - I think the issue might be that the chart only applies the `securityContext` to the FAH container, not the alpine container.  I'll test this theory...

---

Ah my mistake. I forgot and just assumed the docker image is root by default. I'll take a look at your other pr but I think you're right. I'll also test it in my cluster.